docs/v3/tutorials/platform.mdx

```diff
@@ -0,0 +1,195 @@
+
+## Create work pools
+
+Work pools let you dynamically provision infrastructure for your flows.
+First, create a work pool for your production workspace.
+
+1. Switch to the `production` workspace using the workspace switcher in the sidebar.
+1. Click **Work pools** in the sidebar.
+1. Create a new **Docker** hybrid work pool, named `default-work-pool` and using the default configuration.
+
+Next, repeat this steps and create a second work pool named `default-work-pool` in the `staging` workspace.
+You should now have one work pool in each workspace.
+
+## Start workers
+
+Because you're using hybrid work pools, you need to start at least one worker for each work pool to process flow runs.
+
+First, you need to authenticate to Prefect Cloud using the CLI.
+
+```bash
+prefect cloud login
+```
+
+Once you've authenticated to Prefect Cloud, open two terminal windows and run two workers:
+
+```bash Terminal 1
+# Switch to the production workspace
+prefect cloud workspace set --workspace "<account>/production"
+
+# Start a worker for the production work pool
+prefect worker start --pool default-work-pool
+```
+
+```bash Terminal 2
+# Switch to the staging workspace
+prefect cloud workspace set --workspace "<account>/staging"
+
+# Start a worker for the staging work pool
+prefect worker start --pool default-work-pool
+```
+
+<Note>
+You can use `prefect cloud workspace ls` to see the fully qualified names of the workspaces in your account
+</Note>
+
+## Deploy and run flows
+
+Run the following script from the demo repository to create flow runs in each workspace:
+
+```bash
+# Run flows in the production workspace
+prefect cloud workspace --set "<account>/production"
+python simulate_failures.py
+```
+
+```bash
+# Run flows in the staging workspace 
+prefect cloud workspace --set "<account>/staging"
+python simulate_failures.py --fail-at-run 3
+```
+
+Each script invocation will take approximately one minute to finish.
+If you check the logs for the worker running in the staging workspace, you will see errors, this is expected.
+
+## View flow run activity
+
+Now that you've deployed and run your flows, you can view the run details in the UI.
+If you go to the **Home** page for each workspace, you can see run activity.
+
+![Home page](/v3/img/ui/home-page.png)
+
+Here's what happened:
+
+- The script created two deployments of the flow, one deployment per workspace.
+- The script then generated multiple flow runs for each deployment.
+- The activity graph on the **Home** page shows successful runs as green and failed runs as red.
+- Notice that the flow runs in the `production` workspace succeeded, but some of the flow runs in the `staging` workspace failed.
+
+This output is expected, you'll learn how to debug these failures in the next tutorial.
+
+## Optional: Manage team access
+
+If you're a Prefect Enterprise customer, you can use Teams to manage access to your workspaces and work pools.
```

Comment:
Consider linking to teams docs page/section here or below.

Reply:
https://github.com/PrefectHQ/prefect/pull/16123/commits/f38f3782236ea0a3526e78ddb8cb44d6834c2524